dashboard/app/models/census/README.md

```diff
@@ -62,7 +62,6 @@ Summaries are recomputed every night via a [cron job](https://github.com/code-do
 
 # Mapping from `census_submissions` column names to census questions
 Topic questions only show up for those who indicated that their school has a 10 or 20 hr class.
-INACCURACY_REPORTED and INACCURACY_COMMENT only show when the user clicks the "I believe that the categorization for this school is inaccurate" link from the map info window.
```

Comment:
can also remove lines 40-49?

Reply:
That's already being removed in this PR: https://github.com/code-dot-org/code-dot-org/pull/50274